lib/dropRepeats/index.js

```diff
@@ -0,0 +1,10 @@
+import equals from '../equals'
+
+export default function dropRepeats(list) {
+  var result = []
+  for (var i = 0; i < list.length; i++) {
+    if (equals(result[result.length - 1], list[i])) continue
```

Comment:
isn't faster and shorter to compare to `list[i - 1]`?

Reply:
definitely